Homework/HW2/HW2 Programming/src/db_test.py

```diff
@@ -0,0 +1,93 @@
+import unittest
+
+from db import DB
+
+class DBTest(unittest.TestCase):
+    def run_test_table(self, func, tests):
+        for args, want in tests:
+            self.assertEqual(func(*args), want)
```

Comment:
assertEqual (unfortunately) wants (expected, actual). I opened PR https://github.com/donald-f-ferguson/W4111-Intro-to-Databases-Spring-2024/pull/3.